api/app.js

```diff
@@ -53,14 +59,18 @@ if (config.get('graphql.graphiql')) {
     context: { req, res }
   })));
 
-  app.use('/graphiql', graphiqlExpress({ endpointURL: '/graphql' }));
+  app.use('/graphiql', graphiqlExpress({
+    endpointURL: '/graphql',
+    subscriptionsEndpoint: `ws://localhost:3001/subscriptions`
```

Comment:
@nazar  does this need to come from a config value?

Reply:
definetly